package.js

```diff
@@ -8,8 +8,8 @@ Package.describe({
 
 Package.onUse(function(api) {
   api.versionsFrom('1.2.1');
-  api.imply('fourseven:scss@4.5.4');
-  api.use(['ecmascript', 'jquery', 'fourseven:scss@4.5.4'], 'client');
+  api.imply('fourseven:scss@3.4.1 || 4.0.0');
```

Comment:
I do not yet understand this range.

Reply:
`fourseven:scss@4.5` is only compatible with Meteor >= v1.4.1. See https://github.com/fourseven/meteor-scss. Using `3.4.1 || 4.0.0` tell Meteor to choose the version he want `>=3.4.1` (minor and patch versions are automatically considered as compatible), which will be compatible with the client Meteor version from from Meteor `v1.2`).

Reply:
Ah ok, so >= is implied then.